backend/app/helpers/spree/admin/navigation_helper.rb

```diff
@@ -33,7 +33,7 @@ def admin_page_title
         elsif content_for?(:page_title)
           content_for(:page_title)
         elsif admin_breadcrumbs.any?
-          admin_breadcrumbs.map{ |x| strip_tags(x) }.reverse.join(' - ')
+          admin_breadcrumbs.map{ |breadcrumb| strip_tags(breadcrumb) }.reverse.join(' - ')
```

Comment:
What about taking advantage of the change on this line and also add a space after `map` for consistency?

Reply:
Sure, good call. 👍